Add note about SAML auto login (#61926)
grafana/grafana: docs/sources/whatsnew/whats-new-in-v9-4.md

For example, you could assign a user in your organization, whom has the Viewer basic role (note that a user must still
have a basic role assigned) the new Grafana OnCall RBAC role of "Schedules Editor". This would allow the user to view
everything in Grafana OnCall, and also allow them to edit OnCall Schedules

## SAML auto login

> We're added auto login feature support for SAML authentication. It can be turned on with `auto_login` configuration option. OAuth

Reviewer: ```suggestion
We've added auto-login support for SAML authentication, which can be turned on with the `auto_login` configuration option. OAuth
```

> We've added auto-login support for SAML authentication, which can be turned on with the `auto_login` configuration option, and now

Reviewer: ```suggestion
We've added auto-login support for SAML authentication, which you can turn on with the `auto_login` configuration option. We also 
```

> auto login feature was also revamped to have a unified configuration style among all authentication providers. Instead of

Reviewer: ```suggestion
have a unified configuration style among all authentication providers. Instead of using 
```

> `oauth_auto_login`, use the new `auto_login` option enable automatic login for specific OAuth providers.

Reviewer: ```suggestion
`oauth_auto_login`, use the new `auto_login` option to enable automatic login for specific OAuth providers.
```

> `oauth_auto_login`, new `auto_login` option should be used to enable automatic login for specific OAuth provider.

Reviewer: ```suggestion
`oauth_auto_login`, use the new `auto_login` option enable automatic login for specific OAuth providers.
```
